### ISSUE TITLE
Parameters for Bi-LSTM + CRF, Bi-LSTM + CRF + external word embedding news

### ISSUE THREAD

**Issue by pvcastro** [deepmipt/ner#3]
Hello there, great work!

Could you please specify the exact parameters that should be used in order to replicate your experiments from the paper, for both "Bi-LSTM + CRF" and "Bi-LSTM + CRF + external word embedding news" models? The model_params for instantiating the NER network and the learning_params for training.

Thanks!

**Comment by mshnmshn** [comment]
Following the paper mentioned in the README.md (https://arxiv.org/pdf/1709.09686.pdf) the parameters are:

– Word embedding dimension: 100
– Char embedding dimension: 25
– Dimension of hidden layer: 100 (for each LSTM: forward layer and backwardlayer)
– Learning method: SGD, learning rate: 0.005
– Dropout: 0.5
– Number of sentences: 2136 (for training/ validation/ testing: 128 2/ 427/ 427)
– Number of words: 25372 (unique words: 7876). 7208 words (account for 91.52% of unique words) was initialized with pre-trained embedding Lenta
–epochs: 100

**Comment by pvcastro** [comment]
Great, thanks @mu-arkhipov !
